packages/dashboard/src/portal/components/FarmNodesTable.vue

```diff
@@ -850,7 +850,7 @@ export default class FarmNodesTable extends Vue {
     }
   }
   domainCheck() {
-    if (this.domain == "") return false;
+    if (this.domain == "") return true;
```

Comment:
is there a point for having the condition? I think we can remove the whole line

Reply:
I keep it to prevent url validation to return false; will test what will happen if we remove it

Reply:
I tested it, removing this condition makes the field invalid